app/manager/settings/page.jsx

```
@@ -0,0 +1,4 @@
+// return a empty page
+export default function SettingsPage() {
+  return <div></div>;
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)